python/cuda_parallel/cuda/parallel/experimental/__init__.py

```diff
@@ -219,7 +221,8 @@ def __call__(self, temp_storage, d_in, d_out, init):
             d_temp_storage = None
         else:
             temp_storage_bytes = ctypes.c_size_t(temp_storage.nbytes)
-            d_temp_storage = temp_storage.device_ctypes_pointer.value
+            # see comment in _device_array_to_pointer
```

Comment:
The comment is just 2 lines, so lets copy
```suggestion    
            # Note: this is slightly slower, but supports all ndarray-like objects as long as they support CAI
            # TODO: switch to use gpumemoryview once it's ready
```